### ISSUE TITLE
Abnormal circulating Tetranectin concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6007]
**Preferred term label:** Abnormal circulating Tetranectin concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of tetranectin in the blood is outside the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating Tetranectin concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of tetranectin in the blood is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Tetranectin concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating Tetranectin concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of tetranectin in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Tetranectin concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is not a clinical test that has a normal range such that an individual can display an "abnormal" value for it. A good way of deciding whether a lab test is a valid HPO term is to check whether there is a LOINC code for the corresponding test. If there is, that means that it is being tested clinically in at least some situations.
